### ISSUE TITLE
Kolibri Onboarding: MVP Keyboard Accessibility

### ISSUE THREAD

**Issue by marcellamaki** [learningequality/kolibri#13535]
<img height="20px" src="https://i.imgur.com/c7hUeb5.jpeg">

❌ **This issue is not open for contribution. Visit <a href="https://learningequality.org/contributing-to-our-open-code-base/" target="_blank">Contributing guidelines</a>** to learn about the contributing process and how to find suitable issues.

<img height="20px" src="https://i.imgur.com/c7hUeb5.jpeg">

## Description

This PR is narrowly scoped to do an initial accessibility improvement for the first workflow of the Kolibri onboarding project and establish the pattern for subsequent workflows. This issue does _not_ include further enhancements to support screen reader users, but rather lays the foundational pieces for both keyboard and screenreader users in terms of navigation and properly associating elements together. 

### Current state
In the current implementation, you can see that the focus outline (a visible indicator of the focus when using tab navigation) moves behind the onboarding flow. 

https://github.com/user-attachments/assets/e9e2e44f-b09a-4bf3-aa8a-3bb163ca1579

We will need to set the focus intentionally on each element as part of the onboarding flow.

### Requirements
- The [focus](https://www.w3.org/WAI/WCAG21/Understanding/on-focus.html) should move predictably throughout the onboarding flow, and match the visual experience. 

<img width="500" alt="Image" src="https://github.com/user-attachments/assets/7ede5126-154f-4fc1-827c-fbd93affe794" />
<img width="500" alt="Image" src="https://github.com/user-attachments/assets/f009ea73-99cc-48a2-86c4-783323bfbb71" />
<img width="500" alt="Image" src="https://github.com/user-attachments/assets/ebd2cc3b-c5c2-44d6-b7f9-94380984ae27" />
<img width="500" alt="Image" src="https://github.com/user-attachments/assets/af028b6c-47d3-47e5-8473-6a97e62e9c8a" />

- Between each modal, the focused element should be the "continue" button of the following modal. (For users navigating by keyboard, this means the focus outline should be visible).

https://github.com/user-attachments/assets/dd0971a2-a2a8-40ba-ac58-85aa1d04d5b7

- The focus should be trapped between the "X" (close) and "Continue" buttons, meaning the user should not be able to "escape" the modal unless they click the close or continue button.
- When the onboarding tour is active and the overlay is present, the user should not be able to scroll the page

- A title should be added to `TooltipContent` that will be visually hidden. This text will add context on which page and what step the onboarding is on. The global class [.visuallyhidden](https://github.com/learningequality/kolibri/blob/41aff1d3ba20c5022480de9a3dfcc82493380ccb/packages/kolibri/styles/internal/main.scss#L64) should be set on this text so that it is visible to screen readers, but not displayed as content within the tooltip.
- A new string `onboardingStepDescription` for this title should be added to kolibriOnboardingGuideStrings.js with the message: "{pageTitle} - Onboarding - step {currentStep} of {totalSteps}" and passed into `TooltipContent`. 
   - For the first step on the library page, the output of this message should be: “Library - Onboarding - step 1 of 4”

### Additional resources and references
- https://webaim.org/techniques/keyboard/
- [an example of where we are manually setting the focus](https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/coach/assets/src/views/common/assignments/SidePanelRecipientsSelector/LearnersSelectorSidePanel.vue#L149)
- [Tab index information](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/tabindex)
-  [Example](https://github.com/learningequality/kolibri/blob/a8e3376ef3f402ab3f45a5f4348a0c2a738310fb/packages/kolibri-common/strings/enhancedQuizManagementStrings.js#L145) of how strings are defined with props and [how the prop is passed](https://github.com/learningequality/kolibri/blob/a8e3376ef3f402ab3f45a5f4348a0c2a738310fb/kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/CreateQuizSection.vue#L229) to the string in Kolibri

**Comment by LianaHarris360** [comment]
Closed by #13552
